### PR TITLE
2025 Rules Updates

### PR DESCRIPTION
**Pull request by barm**

### CODE REVIEW

File: rules.typ [hunk 1]
```diff
@@ -12,69 +12,58 @@ Welcome to 86 Challenge. Drivers of all skill levels are welcome and there is no
...
-    + If a driver participates in a dedicated 86 Challenge run group, their timing will be taken exclusively from that run group.
-
-  + *Classes*
-    + Cars will be classed according to rules and modification points listed in the following sections.
-    + A car may be used in multiple classes if driven by different drivers. Please comment on your event registration form if you are co-driving a car and what group you are driving in.
-    + Prior to a driver’s first event of the season, they may declare to run in a higher class than their modification points would normally dictate.
-    + A driver shall not compete in more than one class during an event.
-    + If a driver modifies their car mid-season and moves up or down a class, the points earned in the previous class will NOT be carried over to the new class. Drivers may choose to not earn points in the new class and keep the points in the previous class. Drivers participating in the GT Radial Championship Prize challenge cannot change classes.
+    + #highlight[*Tiebreaker*: If competitors have identical times in an event, times set earlier in the day rank above times set later]
```
Comment (xeron) [inline rules.typ:58]: Earlier by what? Actual time? Session number? How would it work for people in different run groups?

File: rules.typ [hunk 2]
```diff
@@ -199,32 +220,33 @@ Questions, corrections, and clarifications can be raised on our #link("https://w
...
-    + OEM-style, off-the-shelf, non-adjustable, and non-rebuildable dampers with unaltered valving are allowed
     + Braking upgrades are allowed
-    + Replacing OEM parts with newer/older OEM parts is allowed (e.g.\ dampers, sway bars, etc).
-    + No lightweight replacement parts are allowed (e.g.\ battery, lighter than OEM trunk) or removal of parts allowed.
-    + Removal of spare tire, floor mats, and tools is allowed.
-    + Non-OEM cat-back exhaust with a weight greater than 32 lbs is allowed (no removal of any catalyst)
+    + Replacing OEM parts with newer/older OEM parts is allowed (e.g. dampers, sway bars, etc)
+    + No lightweight replacement parts are allowed (e.g. battery or lighter-than-OEM trunk) or removal of parts allowed
+    + Removal of spare tire, floor mats, and tools is allowed
+    + OEM-style, non-rebuildable, off-the-shelf dampers are allowed as long as they have: non-adjustable damping, non-adjustable ride-height, and unaltered valving
```
Comment (xeron) [inline rules.typ:239]: Remove `non-rebuildable`.

File: rules.typ [hunk 3]
```diff
@@ -303,160 +326,207 @@ Questions, corrections, and clarifications can be raised on our #link("https://w
...
+      #columns(2)[
+        - Continental ExtremeContact Sport (ECS)
+        - Dunlop Direzza ZII Star Spec
+        - Dunlop Direzza ZIII
+        - Falken Azenis RT615K
+        - Federal 595 RS-RR
+        - Goodyear Eagle F1 Supercar 3
+        - Hankook Ventus RS-4
+        - Kumho Ecsta V730
+        - #highlight[Maxxis VR-1]
```
Comment (xeron) [inline rules.typ:357]: Add `Victra` for consistency.

File: rules.typ [hunk 4]
```diff
@@ -303,160 +326,207 @@ Questions, corrections, and clarifications can be raised on our #link("https://w
...
 #table(
   table.header([Points], [Modifications]),
 
-  [0], [Really Small Rear Aero: OEM rear spoilers, OEM wings, aftermarket \<60mm height],
-  [0.5], [Small Rear Aero: Aftermarket rear spoiler >60mm height],
-  [1], [Medium Rear Aero: Gen. 1 BRZ tS wing, SARD LSR, wings with max width of 1400mm],
-  [1.5], [Large Rear Aero: APR GTC-200, Voltex Type 12, wings with max width of 1460mm],
-  [2], [Unlimited Rear Aero: Wings wider than 1460mm, higher than 245mm, or chord length >250mm],
-  [0.5], [Any rear diffuser extending in front of the rear axle or behind the rear bumper],
+  [0.5], [Canards, dive planes, hood vents, fender vents, and/or vertical deviations on splitters],
```
Comment (xeron) [inline rules.typ:443]: Maybe combine this with underbody and name the group as "Other Aero"?

**Comment by xeron** [comment]
Can we try Arial font for the rules? Serif is kinda annoying.

File: rules.typ (hunk 1 above)
Comment (barm) [reply rules.typ:58]: Wall time is what I was thinking.

File: template.typ [hunk 5]
```diff
@@ -42,17 +47,31 @@
 
 #let spacer = block(above: 1.25em, below: 1.25em)
 
-#let frontmatter(title: str, revision: str) = [
+#let website = link("https://86challenge.us")[86challenge.us]
+
+#let facebook = link("https://www.facebook.com/groups/1009986312348877")[Facebook Group]
```
Comment (xeron) [inline template.typ:52]: Should we use https://www.facebook.com/groups/86challenge instead?

File: rules.typ (hunk 1 above)
Comment (barm) [reply rules.typ]: Updated

File: rules.typ [hunk 6]
```diff
@@ -412,57 +411,55 @@ _Any tire not on this list is automatically 6pts_
...
 == Front Aero
 
 _Take points for ONE of the following:_
 
 #table(
   table.header([Points], [Modifications]),
-  [0], [
-    Any device extending less than 50mm forward/outward/rearward from the bottom leading edge of the bumper (including STI lip and TRD lip)
-  ],
+  [0], [No aero modifications or any device extending less than 50mm forward/outward/rearward from the bottom leading edge of the bumper (including STI lip and TRD lip)],
```
Comment (xeron) [inline rules.typ]: Same for the rear?

File: rules.typ [hunk 7]
```diff
@@ -303,160 +346,198 @@ Questions, corrections, and clarifications can be raised on our #link("https://w
...
+
 #table(
   table.header([Points], [Modifications]),
-
-  [0], [OEM-style, non-adjustable dampers],
-  [0.5], [Non-OEM springs],
-  [0.5], [Adjustable dampers or modification to damper valving],
-  [0.5], [Remote reservoirs],
+  [0], [
+    OEM rear spoiler, OEM wing (except Gen. 1 BRZ tS wing), or any aftermarket spoiler less than 60 mm in height
```
Comment (xeron) [inline rules.typ:475]: Add no aero here too.

**Review by xeron — APPROVED** [review]

**Pull request merged**